dev/com.ibm.ws.jaxrs.2.0_fat/test-applications/options/src/com/ibm/ws/jaxrs/fat/options/OptionsApplication.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxrs.fat.options;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+@ApplicationPath("options")
+public class OptionsApplication extends Application {
+
+    @Override
+    public Set<Class<?>> getClasses() {
+        Set<Class<?>> classes = new HashSet<Class<?>>();
+        classes.add(OptionsResource.class);
+        classes.add(OptionsResource2.class);
+        classes.add(OptionsTestServlet.class);
+        return classes;
+    }
```

Comment:
We can delete this code and let the resource classes be auto-discovered.  If you wanted to save more code, you could make one of the resource classes extend Application and add the `@ApplicationPath` to it.  Either way.

```suggestion
```